src/pages/index.js

```diff
@@ -37,61 +60,106 @@ profileEditButton.addEventListener("click", () => {
   editProfilePopup.open();
 });
 
+const editProfilePopup = new PopupWithForm(profileEditModal, (data) => {
+  editProfilePopup.saving(true);
+  api.updateUserInfo(data).then((data) => {
+    userInfo.setUserInfo(data.name, data.about);
+    editProfilePopup.saving(false);
```

Comment:
`    editProfilePopup.saving(false);
`

You need to return the **default** button text only in blocks `finally` so that the button could get the text back in any case: `success` or `failure`.


**Please, correct it everywhere**